test/error-placement.js

```diff
@@ -311,6 +311,21 @@ test( "test existing non-error aria-describedby", function( assert ) {
 	strictEqual( "", $("#testForm17text-error").text(), "Error label is empty for valid field" );
 });
 
+test( "test existing non-error aria-describedby with bracket", function( ) {
+	expect( 2 );
+	var form = $( "#testForm18" ),
+		field = $( "#testForm18\\[text\\]");
+
+	field.attr("aria-describedby", "testForm18\\[text\\]-description testForm18\\[text\\]-error");
```

Comment:
I understand your position and your right.
I did it that way because using .valid() do not reproduce the problem I got by checking/unchecking a required checkbox. .element() don't reproduce it neither. After many hours looking around I had just given up. I'm sorry but I will have to dig deeper into the plugin to understand why I'm not able to simply reproduce it with common method.